### DISCUSSION TITLE
Change DemiBold behavior from `11.0.x` to `11.1.0`

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#16484] [Q&A] [answered]
Considering the following xaml

## XAML

```xaml
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="AFontIssue.MainWindow"
        Title="AFontIssue">
  <TextBlock Text="📐 Projects"
             FontSize="54"
             FontWeight="DemiBold"
             />
</Window>
```

## Avalonia `11.0.12` behavior
![Avalonia 11.0.12](https://github.com/user-attachments/assets/9c9f1d15-d3cf-4393-be28-adb77b45e61b)

## Avalonia `11.1.0` behavior
![Avalonia 11.1.0](https://github.com/user-attachments/assets/6709c27c-d7ce-498e-8084-6344799b20f1)

Platform: **Window 10 x 64**

What is the expected behavior?

I attach a sample project. To simulate the problem, just change the `AvaloniaVersion` property in `AFontIssue.csproj`.

[AFontIssue.zip](https://github.com/user-attachments/files/16389510/AFontIssue.zip)

**Comment by Gillibald** [comment]
So the issue is that a regular glyph is rendered instead of a color glyph?

**Reply by workgroupengineering** [reply]
Yes

**Comment by rabbitism** [comment]
same happens to semi avalonia demo.

**Comment by Gillibald** [comment]
I think this is because of bold emulation. There is no Bold/DemiBold variant and therefore the backend emulates the typeface. This seems to remove color glyphs.

**Comment by Gillibald** [comment, marked as answer]
Should be fixed by: https://github.com/AvaloniaUI/Avalonia/pull/16489
